nmmo/core/observation.py

```diff
@@ -193,8 +192,9 @@ def _make_action_targets(self):
 
   def _make_move_mask(self):
     # pylint: disable=not-an-iterable
+    habitable_material = set(material.Habitable)
```

Comment:
we could compute this in the constructor. alternatively, maybe we just make material.Habitable into a set?

Reply:
Seems like Habitable is already a set in the constructor so this is redundant. I'll raise a new PR without this change.